src/Slide.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Slide = ({ src, children }) => {
-  if (typeof src === 'string') {
+const Slide = ({ image, children }) => {
+  if (typeof image === 'string') {
     return (
       <li>
-        <img src={src} alt="" />
+        <img src={image} alt="" />
         {children}
       </li>
     );
   } else {
     return (
       <li>
-        {src}
+        {image}
         {children}
```

Comment:
Sorry for this, but I still don't see how this could work, we would render an image next to the children? Can you point me to a materializecss example where this happens?

Reply:
Well, you will not find that in the documentation. 
You can have a look [here](https://github.com/Dogfalo/materialize/blob/v1-dev/js/slider.js#L58) to understand why this is going to work with a `picture` element.
Basically this allows one to use a [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) html element for [art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#Art_direction) or simply to provide different version of your images (WebP images, JPEG...)
For the full example please see my first comment.

Reply:
@alextrastero Let me know if you still want to use `src` as a prop name. I think `image` or `img` would be better. Or maybe having it both?
Like `src` is a `string` and it will handle the first case and `image` is a `node` and we are going to render it differently.
Basically it's the same thing is happening now under the hood, but doing so will not break existing code.